### ISSUE TITLE
Can't open plugin menu

### ISSUE THREAD

**Issue by gamester26** [Jared0714/Multi-PokemonFramework#13]
Hello! I have the latest version of Luma, (13.0.2), and the latest version of Multi-pokemon framework. When trying to open the plugin menu in-game after installing, nothing happens.
I do get the blue screen flash when booting up the games however.

**Comment by urcoolyo** [comment]
I’m having the exact same problem unfortunately.

**Comment by KILLERsandwich** [comment]
I have the same problem with pokemon ultra moon but only when is use the ROM hack prismatic moon via the luma game paching Funktion, without patching the ROM hack to the game, the menu opens up normally so it isn't compatible with ROM hacks I guess? Which is pretty unfortunate

**Comment by biometrix76** [comment]
The plugin was never meant to support ROM hacks, that would be very tedious due to the sheer amount of versions out there

**Comment by Ninjistix** [comment]
Just tested this today as of Luma v13.3.3

DL latest release and extract it to sd card
Open up the Rosalina menu by pressing L + Down + Select
Scroll down to Plugin Loader and press A to enable it
Launch game and you should see the screen flash blue
Once you see Welcome in Black n White Text Bottom right corner of top screen
Press Select to open plugin menu. 


(note the welcome text will not show up unless you have already created a char and started the game)
Tested on the following games
Pokemon X
Pokemon Y
Pokemon OR
Pokemon AS
Pokemon Sun
Pokemon Moon
Pokemon US
Pokemon UM

**Comment by biometrix76** [comment]
Yeah I guess the check could be improved upon. Was mostly to prevent users from opening during loading/title screens of the game(s) as it was known to cause freezing. This version of the plugin is outdated now as I've lost access to this account and therefore could not publish changes and commits. Updated and separated plugins could be found elsewhere now

**Comment by Ninjistix** [comment]
are their updated plugins somewhere? i haven't checked in awhile idk.
